spec/spec.md

```diff
@@ -123,6 +123,55 @@ Database |
 Finalize the component stack list - are these correct? Are we missing any?
 :::
 
+## Addressing
+
+A user's logical Identity Hub (their collection of Hub Instances) can be addressed in many ways, 
+but the mechanisms below ****MUST**** be supported by a compliant Identity Hub implementation:
+
+### DID-Relative URLs
+
+The following DID URL constructions are used to address [[ref: Hub Instances]] found to be associated 
+with a given DID, as located via the DID resolution process.
+
+*DID-Relative URL addressing a single file:*
+
+```json
+did:example:123?service=IdentityHub&resource=Qm3fw45v46w45vw54wgwv78jbdse4w
```

Comment:
Do we now need to add support for this in every DID method? Or how will the resolution process work here?

Reply:
Kinda depends what happens with the Service resolution aspects of DID Core, because they are currently marked at-risk. I am not sure whether it really matters much in the end, as the URLs will be constructed either way regardless. If it swings to Methods or standard lib code doing it, I don't think it's a big loss.